Makefile

```diff
@@ -46,6 +46,8 @@ install:
 	install -d -m 755 "$(DESTDIR)"/usr/lib/systemd/{system,system-generators}
 	install -d -m 755 "$(DESTDIR)"/usr/lib/tmpfiles.d
 	install -m 644 systemd/openqa-worker@.service "$(DESTDIR)"/usr/lib/systemd/system
+	install -m 644 systemd/openqa-worker@.service "$(DESTDIR)"/usr/lib/systemd/system/openqa-worker-no-cleanup@.service
+	sed -e 's_^\(ExecStart=/usr/share/openqa/script/worker\) \(--instance %i\)$_\1 --no-cleanup \2_' "$(DESTDIR)"/usr/lib/systemd/system/openqa-worker-no-cleanup@.service
```

Comment:
I don't think this does what you want to achieve. sed -e only outputs the changed file to STDOUT.

You either add a -i to the mix - or leave out the install completely and sed -e ... > "$(DESTDIR)"/...